CONTRIBUTING.md

```diff
@@ -10,33 +10,73 @@ please read the [code of conduct](CODE_OF_CONDUCT.md).
 ```sh
 $ git clone https://github.com/netlify/netlify-cms
 $ cd netlify-cms
-$ yarn
+$ yarn start
 ```
 
-## Building
+## Available scripts
+
+### `bootstrap`
+
+Installs `yarn` dependencies and bootstraps the `lerna` packages.
```

Comment:
Let's remove the references to Yarn and Lerna here -- people can look up implementation details if they want to, and it may be confusing if they aren't familiar.

Maybe something like:
> Installs and bootstraps CMS package dependencies.

Reply:
cc/ @verythorough